Makefile

```diff
@@ -9,3 +9,12 @@ docs:
 	pip install -r requirements.txt &&\
 	make html &&\
 	cd -
+
+dev-setup:
+	pip install -e ".[test]"
```

Comment:
These commands are going to conflict with https://github.com/graphql-python/graphene/pull/971/files#diff-b67911656ef5d18c4ae36cb6741b7965